stdlib/2/__builtin__.pyi

```diff
@@ -44,6 +45,8 @@ class object:
     def __getattribute__(self, name: str) -> Any: ...
     def __delattr__(self, name: str) -> None: ...
     def __sizeof__(self) -> int: ...
+    def __reduce__(self) -> Union[str, tuple]: ...
```

Comment:
@ambv You're right. `object.__reduce__` always returns a `tuple` unless it's overridden in inheritors. So inheritors are free to specify `str` or `tuple` or a specific tuple as the actual return type of the override.